src/components/ButtonGroup/ButtonGroup.stories.tsx

```diff
@@ -4,7 +4,7 @@ import { Button } from '../Button/Button'
 import { Link } from '../Link/Link'
 
 export default {
-  title: 'ButtonGroup',
+  title: 'Components/Button groups',
```

Comment:
Minor question: I noticed most of the time `title` follows the convention `Components/ComponentName`, but a few (like this one) use `Components/Component name`.

Is it worth aligning that across all stories?

Reply:
@brandonlenz good call, I tried to match the navigation used in the sidebar on https://designsystem.digital.gov/components/

My thinking was that someone using this lib is probably coming from USWDS and wants to be able to easily find our components from that perspective (and I figure they can find the name of the component itself using the docs or source code)

Reply:
Got it. That makes sense to me then!